datafusion/core/src/physical_plan/file_format/parquet/row_groups.rs

```diff
@@ -263,6 +263,15 @@ mod tests {
     };
     use std::sync::Arc;
 
+    struct PrimitiveTypeField<'a> {
+        name: &'a str,
```

Comment:
I don't think it is important but given that `name` seems to always be a constant string, you could do something like

```suggestion
        name: &'static str,
```

And avoid having to have the named lifetime `'a` on this struct